src/components/DetailsMetrics/detailsMetrics.utils.js

```diff
@@ -130,6 +154,17 @@ export const generateMetricsItems = metrics => {
     .value()
 }
 
+export const getDateRangeBefore = range => {
+  const rangeDuration = range.end - range.start
+  const newStart = range.start - rangeDuration
+  const newEnd = range.end - rangeDuration
```

Comment:
you don't need `newStart` and `newEnd` variables

Reply:
done.